src/components/dashboard-page.tsx

```
@@ -12,6 +15,10 @@ export function DashboardPage() {
   const { chainId } = useAppKitNetwork();
   const { successMessage, errorMessage } = useStatusMessageState();
 
+  const { successMessage, errorMessage, setErrorMessage, clearMessages } = useStatusMessage();
```

Biome (2.3.13)

[error] Shouldn't redeclare 'successMessage'. Consider to delete it or rename it.

'successMessage' is defined here:


(lint/suspicious/noRedeclare)